src/prototype.rs

```diff
@@ -633,10 +647,12 @@ fn parse_blocks(mut tree: &mut Tree<Item>, s: &str, mut ix: usize) -> usize {
 
     let (num_code_fence_chars, code_fence_char) = scan_code_fence(&s[ix..]);
     if num_code_fence_chars > 0 {
+        let nextline_offset = scan_nextline(&s[ix..]);
+        let info_string = s[ix+num_code_fence_chars..ix+nextline_offset].trim().to_string();
```

Comment:
I _think_ this needs to be unescaped (this is done in the master branch), but I haven't double-checked with a test. (this is the kind of thing that arguably should be covered in the spec test, and if it isn't we should add one).